tests/debug/README.md

```
@@ -60,3 +60,44 @@ One JSON object per OOB event, one event per line:
+The open question it's waiting to answer: does a Gen3 dongle answer solicited
+reads at all, or *only* fan out? (No Gen3 capture yet.)
+
+### Usage
+
+```bash
+uv run python tests/debug/unsolicited_responses.py CAPTURE.log [CAPTURE2.log ...]
+```
+
+### Interpreting the output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)